pyproject.toml

```diff
@@ -112,18 +112,36 @@ exclude_lines = [
 [tool.tox]
 legacy_tox_ini = """
 [tox]
+envlist = lint,pytest
 isolated_build = True
-envlist = site
+labels =
+  venv = py311-lint, py3{6,7,8,9,10,11,12}-pytest
```

Comment:
This list will get out of date quickly. Is there some way to tell it "all versions > min that you support"?

Reply:
I wasn't able to find anything... I also want to explicitly make sure we're getting `py36-pytest` included...

Reply:
Ack, thanks. Resolving.